mlocal/frags/Makefile.stub

```diff
@@ -226,6 +226,17 @@ check: $(BUILDDIR)/vendors-done
 	$(V)cd $(SOURCEDIR) && golint -set_exit_status `go list ./...`
 	@echo "       PASS"
 
+.PHONY: dist
+dist: $(BUILDDIR)/vendors-done
+	@echo " DIST setup VERSION"
+	$(V)git describe --match 'v[0-9]*' --dirty --always > $(SOURCEDIR)/VERSION
```

Comment:
Should not manually be doing `git describe` again. The `$(VERSION)` variable has what we need already

`$(V)echo $(VERSION) > $(SOURCEDIR)/VERSION`